.env

```diff
@@ -0,0 +1 @@
+PORT=3001
```

Comment:
you also shouldn add the .env file into .gitignore, and add a new .env.template that people can clone